protos/channel_responses.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+/* Following chromium style (e.g. sync_pb) is the usage for the sync protobuf
+*/
+package publishers_pb;
+
+message SocialLinks {
+  string youtube = 1;
+  string twitter = 2;
+  string twitch = 3;
+}
+
+message SiteBannerDetails {
+  string title = 1;
+  string description = 2;
+  string background_url = 3;
+  string logo_url = 4;
+  repeated int32 donation_amounts = 5;
+  SocialLinks social_links = 6;
+}
+
+enum WalletConnectedState {
+  NO_VERIFICATION = 0;
+  UPHOLD_ACCOUNT_NO_KYC = 1;
+  UPHOLD_ACCOUNT_KYC = 2;
+  /* (Albert Wang) These might be used */
+  PAYPAL_ACCOUNT_NO_KYC = 3;
+  PAYPAL_ACCOUNT_KYC = 4;
+}
+
+message ChannelResponse {
```

Comment:
Question: why do we call them "channels" here but "publishers" elsewhere? Should we call them just one thing?

Reply:
So on the server-side, publisher refers to the person or organization. Channel is the domain representation. I realize it's confusing, so I can make the change from PublisherResponse to ChannelResponse if you desire.

Reply:
No, I think `ChannelResponse` is fine.